website/src/pages/index.tsx

```diff
@@ -194,8 +194,8 @@ function Feature({
       <img
         className={styles.featureImage}
         alt={feature.title}
-        width={feature.image.width}
-        height={feature.image.height}
+        width={parseInt(feature.image.width, 10)}
```

Comment:
`width` is already a number, `Math.floor` is sufficient. Also shall we simply truncate it in the original data?